<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signwind</class>
 <widget class="QDialog" name="signwind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-Voting SignUp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main Container */
QWidget {
    background-color: #f8f9fa;
    font-family: 'Segoe UI', sans-serif;
    padding: 20px;
}

/* Title */
QLabel#title_label {
    font-size: 20px;
    font-weight: bold;
    color: #1a3e8c;
    margin-bottom: 25px;
}

/* Section Headers (Username, Password, etc.) */
QLabel.section_header {
    font-size: 14px;
    font-weight: 600;
    color: #495057;
    margin-bottom: 5px;
}

/* Input Fields */
QLineEdit, QDateEdit {
    background: white;
    border: 1px solid #ced4da;
    border-radius: 6px;
    padding: 10px;
    font-size: 14px;
    margin-bottom: 15px;
    min-width: 250px;
}

/* Input Field Placeholder Text */
QLineEdit::placeholder {
    color: #adb5bd;
    font-style: italic;
}



QPushButton#register_btn:hover {
    background-color: #1765cc;
}

/* Horizontal Line */
QFrame#line {
    border: 1px solid #e9ecef;
    margin: 15px 0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="rightMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>60</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">
    font-size: 20px;
    font-weight: bold;
    color: #1a3e8c;  /* Navy blue */
    margin-bottom: 20px;
</string>
       </property>
       <property name="text">
        <string>Sign Up To  Participate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true"> font-size: 13px;
    font-weight: bold;
    color: #1a3e8c;  /* Navy blue */
    padding: 10px;</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_eidt">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Enter your username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true"> font-size: 13px;
    font-weight: bold;
    color: #1a3e8c;  /* Navy blue */
    padding: 10px;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pass_edit">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password must contain letters and numbers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true"> font-size: 13px;
    font-weight: bold;
    color: #1a3e8c;  /* Navy blue */
    padding: 10px;</string>
       </property>
       <property name="text">
        <string>Cnic Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cnicedit">
       <property name="styleSheet">
        <string notr="true"> background: white;
    border: 1px solid #ddd;
    border-radius: 5px;
    padding: 10px;
    margin: 8px 0;
    min-width: 250px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>Without Dashes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true"> font-size: 13px;
    font-weight: bold;
    color: #1a3e8c;  /* Navy blue */
    padding: 10px;</string>
       </property>
       <property name="text">
        <string>Date Of Birth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>-1</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit {
    color: #000000;  /* Black text */
    background-color: #ffffff;  /* White background */
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>80</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>80</number>
        </property>
        <item>
         <widget class="QPushButton" name="createacc_btn">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #1a3e8c;  /* Navy blue */
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px;
    margin: 10px 0;
}

QPushButton:hover {
    background-color: #0d2b5e;  /* Darker blue */
}</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sign_back">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(229, 62, 62);
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px;
    margin: 10px 0;
}
#sign_back:hover {
    background-color: #c53030;  
}</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
